models/foodprint_qrcode.js

```diff
@@ -61,6 +61,14 @@ module.exports = function (sequelize, DataTypes) {
         type: DataTypes.STRING(255),
         allowNull: true,
       },
+      qrcode_hashid: {
```

Comment:
Hi @DanielmAaron.

No need for this to be a BLOB. It can be a string. We use BLOBS for images/file data.

Also, you will need to add a sequelize migration script for the addition and removal of the new columns
- e.g. `npx sequelize-cli migration:create --name qrcode-columns`

I will merge for now but this will be needed before deploy to PROD